labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+# Accessing Optimizely Enriched Event Data
+
+Optimizely [Enriched Events Export](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) gives you secure access to your Optimizely event data so you can analyze your experiment results with greater flexibility. The export includes a useful combination of events attributes:
+
+- Raw metadata (event names, user IDs, etc) that you pass to Optimizely, without additional processing
+- Enriched metadata that Optimizely adds such as experiment IDs, variation IDs, and session IDs
+
+For more information, see the Enriche Events [data specification](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification).
+
+This Lab contains `oevents`, a simple CLI tool for loading enriched event data.
+
+![oevents demo](img/demo.gif)
+
+## Prerequisites: bash (v4+), date, jq, curl, and aws
+
+`oevents` is written is [bash](https://www.gnu.org/software/bash/) and requires version 4 or greater to run.  It also requires the following prerequisites:
+
+- [`date`](https://www.gnu.org/software/coreutils/manual/html_node/date-invocation.html) (included in OS X and most GNU/Linux distributions)
+- [`jq`](https://stedolan.github.io/jq/)
+- [`curl`](https://curl.haxx.se/)
+- the [Amazon AWS CLI](https://aws.amazon.com/cli/) (v2+) 
+
+## Installation
+
+`oevents` is a bash script. To use it, you can specify the script's path explicitly like this (assuming `oevents` is in your working directory):
+
+```sh
+$ ./oevents help
+```
+
+Or you can add it to a directory in your `PATH` environment variable.  You can do this temporarily by navigating to this tutorial's directory and adding it to your `PATH` variable:
+
+```sh
+$ cd enriched_events/
+$ export PATH="$(pwd):$PATH"
+$ oevents help
+```
+
+## Get Help
+
+For usage instructions, use the `help` command:
+
+```sh
+$ oevents help
+```
+
+## Authenticating
+
+Enriched Event data is served via [Amazon S3](https://aws.amazon.com/s3/).  You can authenticate `oevents` to AWS in two ways:
+
+1. (Recommended) Providing your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) via the `OPTIMIZELY_API_TOKEN` environment variable or the `--token` command line argument. `oevents` will acquire AWS credentials using the [Optimizely Authentication API](https://docs.developers.optimizely.com/optimizely-data/docs/authentication-api).
+
+2. Providing your AWS credentials directly. See the [AWS user guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for instructions.
+
+You can use the `oevents auth` command to acquire temporary AWS credentials:
```

Comment:
Should these instructions be listed under "1. (Recommended)"? since they appear to use the `--token` command line argument?

Reply:
I didn't like the formatting when I put this block in the middle of the ordered list, but I added a "**note**" before it and some clarifying language around the need for a PAT with `oevents auth`.  Hopefully that helps